services/frontend/src/app/AuthenticationLayer.tsx

```diff
@@ -16,21 +16,29 @@ const AuthenticationLayer = ({ children }: Props) => {
   const dispatch = useDispatch();
   const router = useRouter();
 
+  const fetchSessionToken = async () => {
+    const response = await fetch("http://localhost:3000/api/me");
```

Comment:
Should use relative url 

```suggestion
    const response = await fetch("/api/me");
```